gitter/question.yml

```diff
@@ -0,0 +1,4 @@
+when: [question]
+why: [question]
+how: [question]
```

Comment:
and `what`, `where`?

Reply:
Yes, I will add those for sure.

I will add many more. Keep giving your suggestions. ;)